_jobs/033-senior-cloud-automation.html

```diff
@@ -44,7 +48,7 @@ <h3>Required Skills</h3>
                     <li>Understand pros and cons of cloud technologies</li>
                     <li>Experience with <span class="accentuate">Infrastructure as a Service</span> and <span class="accentuate">Platform as a Service</span> technologies.</li>
                     <li>Experience and understanding of the <span class="accentuate">SDLC</span></li>
-                    <li>Strong experience with <span class="accentuate">containerization/orchestration</span> technologies tools like Docker, Kubernetes and optionally Mesos and Swarm</li>
+                    <li>Strong experience with <span class="accentuate">containerization/orchestration</span> technologies tools like Docker, Kubernetes and optionally Mesos or Swarm</li>
                     <li>Strong understanding of <span class="accentuate">GIT</span> version control systems such as GitHub, GitLab or Bitbucket</li>
                     <li>Experience with <span class="accentuate">CI/CD automation tools</span> such as Jenkins, Concourse or Bamboo, and artifact repositories like Nexus or Artifactory</li>
                     <li>Strong experience working in an Agile environment</li>
```

Comment:
Comment for 2 lines below as I think there's a typo in there:
"Strong track record of learning new technology independently" -> "Strong track record of learning new technolog**ies** independently"